docs/helpers/SeleniumWebdriver.md

```diff
@@ -3,21 +3,23 @@
 SeleniumWebdriver helper is based on the official [Selenium Webdriver JS](https://www.npmjs.com/package/selenium-webdriver)
 library. It implements common web api methods (amOnPage, click, see).
 
-#### Selenium Installation
+## Backends
+
+### Selenium Installation
```

Comment:
This section is taken from `lib/helpers/SeleniumWebdriver.js` source code.
It will be regenerated. Please update this text inside the helper itself